libosmscout-import/include/osmscout/import/GenCoordDat.h

```diff
@@ -29,6 +29,15 @@
 
 namespace osmscout {
 
+/**
+ * Some temporary variables used in asserts may be unused
+ * on production build, when asserts are wiped out.
+ *
+ * Usage of this macro for such variables makes production
+ * build happy, and avoids unused-variable warning
+ */
+#define _unused(x) ((void)(x))
```

Comment:
That macro would be a good idea at a more central location. There is a small number of places, where it could be used, too. One question though: It underscore in macros allowed or is this reserved by the standard or similar (similar to two underscores...)